### PR TITLE
Chore/update depends

### PR DESCRIPTION
**Pull request by yoxira**
We can't bump the `chai`, `change-case` and `execa` package versions as they dropped CommonJS support.

### CODE REVIEW

**Comment by yoxira** [comment]
why is `package-lock.json` listed in the `.gitignore` file? 🙂

**Comment by yoxira** [comment]
https://ahead.adamant.im/api/loader/status/sync

**Comment by adamant-al** [comment]
> We can't bump the `chai`, `change-case` and `execa` package versions as they dropped CommonJS support.

For the next version, find alternatives and replace these.

**Comment by bludnic** [comment]
> > We can't bump the `chai`, `change-case` and `execa` package versions as they dropped CommonJS support.
> 
> For the next version, find alternatives and replace these.

Let's migrate to ESM. At some point, we'll have to do it anyway. ESM can still import CommonJS, so we can at least do a partial migration.

**Comment by adamant-al** [comment]
> > > We can't bump the `chai`, `change-case` and `execa` package versions as they dropped CommonJS support.
> > 
> > 
> > For the next version, find alternatives and replace these.
> 
> Let's migrate to ESM. At some point, we'll have to do it anyway. ESM can still import CommonJS, so we can at least do a partial migration.

Agreed. But also next version.

File: package.json [hunk 1]
```diff
@@ -37,67 +37,67 @@
...
+    "strftime": "=0.10.3",
+    "unzipper": "^0.12.3",
     "valid-url": "=1.0.9",
     "validator.js": "=2.0.4",
-    "z-schema": "=5.0.3"
+    "z-schema": "=6.0.2"
   },
   "devDependencies": {
-    "chai": "^4.3.10",
+    "chai": "^4.1.1",
```
Comment (bludnic) [inline package.json:83]: Did you downgrade the version on purpose? Is it because `chai` dropped CommonJS support?

File: helpers/checkIpInList.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 var _ = require('lodash');
-var ip = require('ip');
+var ip = require('neoip');
```
Comment (bludnic) [inline helpers/checkIpInList.js:4]: Why switched to `neoip`? As I see there are no differences in the libs API.

Comment (yoxira) [reply helpers/checkIpInList.js:4]: [chore: replace node-ip with neoip to fix an npm vulnerability](https://github.com/Adamant-im/adamant/pull/82/commits/dd9fef9b0fc9c1d7b7561a20e7326aac28fccdaa)

**Comment by adamant-al** [comment]
> why is `package-lock.json` listed in the `.gitignore` file? :/

Fix it

File: package.json (hunk 1 above)
Comment (bludnic) [reply package.json:83]: @martiliones

**Comment by bludnic** [comment]
Add `package-lock.json` to git. Use Node v18. @martiliones @yoxira

**Comment by adamant-al** [comment]
Can we also update this and merge to `dev`?

File: test/unit/logic/account.js [hunk 3]
```diff
@@ -0,0 +1,156 @@
+"use strict";
```
Comment (bludnic) [inline test/unit/logic/account.js:1]: Do you need this file in that PR? You already added it in #86

https://github.com/Adamant-im/adamant/pull/86/files#diff-a5a19004bec268761189807baa4cd27c71e35301d3f2b6dbeb9867bc82545feb

**Review by bludnic — APPROVED** [review]

**Pull request merged**